app/javascript/angular/code/services/_fixed_sessions.js

```diff
@@ -236,17 +213,14 @@ export const fixedSessions = (
 
       const data = params.get("data");
 
-      // _.values suggests that `params.get('selectedSessionIds')` could be an obj, is it true?
```

Comment:
oh yes, it could, cause `params.get()` might return `{}`: 
```
      get: function(name, defaultValue) {
        return this.paramsData[name] || defaultValue || {};
      },
```
I would suggest we stop using `params.get` and just create direct methods for things we need, then someday we can just move them from params to some vanilla js.